docs/infra/Understanding-Google-Testing.md

```diff
@@ -17,50 +32,62 @@
 
    Once Google Testing finishes, it takes up to 1 hour for the results to be propagated back to Github. Once the result is available on Github, it will show "Google Testing" as either "success" or "failure".
 
+</details>
+
+This check helps catch obvious problems, and allow the pull request author and
+reviewers an early warning that either:
+
+1. More changes are needed to avoid a breaking change
+1. Google code or golden-files will need to be updated as a part of the roll
+1. Some communication with the roll team is required to make the change safely
```

Comment:
```suggestion
1. More changes are needed to avoid a breaking change.
1. Google code or golden-files will need to be updated as a part of the roll.
1. Some communication with the roll team is required to make the change safely.
```